doc/gtdb.md

```diff
@@ -0,0 +1,137 @@
+
+We provide a script: [`genomes_for_db.py`](https://biocore.github.io/wol/code/scripts/genomes_for_db.py) to convert multiple genome sequence files into a single Fasta file.
+
+```bash
+genomes_for_db.py --input input_dir --output output.fna --ext _genomic.fna.gz --concat --gap N*20
+```
+
+This command will removing intra-sequence line breaks, concatenate multiple sequences in the same file (e.g., chromosomes or scaffolds of the same genome) with a linker of 20 "N" characters, and merge all genomes into one multi-Fasta file, in which sequence IDs are genome IDs.
+
+The output file `output.fna` can then be used for database building with choice of tools. The taxonomy files discussed above will match the results of sequence alignment using this database. For examples:
+
+Bowtie2:
+
+```bash
+bowtie2-build --seed 42 --threads 32 output.fna GTDBr89
+```
+
+BLASTn (with taxonomy integrated, see [above](#reformat-gtdb-as-taxdump-style)):
+
+```bash
+makeblastdb -in output.fna -out GTDBr89 -dbtype nucl -parse_seqids -taxid_map taxid.map
+```
+
+Note: The same set of genomes can also be found in the [GTDB-Tk](https://github.com/Ecogenomics/GtdbTk) database, under `fastani/database/`. However these filenames do not have the prefix `RS_` or `GB_`. Therefore, if you choose to use these files, you will need to modify the taxonomy table to match genome IDs:
+
+```bash
+cat taxonomy/gtdb_taxonomy.tsv | sed 's/^RS_\|GB_//' > gtdb_taxonomy_rev.tsv
+```
+
+
+## Use GTDB phylogeny
+
+GTDB is a phylogeny-based taxonomy system, which means that instead of staying with the derived, hard-coded taxonomic ranks, you may directly exploit the original phylogenetic trees. However there are some tricks to be considered:
+
+The "GTDB phylogeny" actually consists of two trees: one for Bacteria ([`bac120_r89.sp_labels.tree`](https://data.ace.uq.edu.au/public/gtdb/data/releases/release89/89.0/bac120_r89.sp_labels.tree)) and the other for Archaea ([`ar122_r89.sp_labels.tree`](https://data.ace.uq.edu.au/public/gtdb/data/releases/release89/89.0/ar122_r89.sp_labels.tree)). In order to use them in one analysis, you need to join them:
+
+(TODO: Add tutorial for using GTDB phylogeny.)
+
+
+## Use WoL database with GTDB taxonomy
+
+If you decide to stick to the WoL genome database (we appreciate that!) but also want to embrace the GTDB taxonomy, we have a read-to-go solution for you:
+
+We provide original and curated NCBI and GTDB [taxonomies](https://biocore.github.io/wol/data/taxonomy/) based on the WoL phylogeny. For GTDB specifically, you may download this file: [`lineages.txt.bz2`](https://github.com/biocore/wol/raw/master/data/taxonomy/gtdb/curation/lineages.txt.bz2), which contains curated GTDB lineage strings for WoL genomes.
+
+Similar to above, you can simply do:
+
+```bash
+woltka classify --lineage lineages.txt.bz2 --rank phylum,genus,species ...
+```
+
+As you would expect, the resulting taxonomic profiles will speak GTDB!
```

Comment:
What do you mean by speak? Does it produce similar output to GTDB? or is it just based on the GTDB lineage?

Reply:
It will be based on the GTDB taxonomy (lineages). Btw, the output table is in a constant format, not in either NCBI or GTDB format.